src/pages/guides/hybrid-plugins/distribute/index.md

```diff
@@ -0,0 +1,16 @@
+# Distribute
+
+Prepare your hybrid plugin to be listed in the Adobe marketplace via [Developer Distribution](https://developer.adobe.com/distribute). To learn more about the platform and general packaging guidelines, visit [here](https://developer.adobe.com/developer-distribution/creative-cloud/docs/guides/getting_started/).
```

Comment:
Refer to it as "the Creative Cloud Marketplace" ... try to keep the wording consistent with: https://blog.developer.adobe.com/introducing-a-new-developer-distribution-portal-for-listing-uxp-plugins-in-the-creative-cloud-7f81376f46a9?source=friends_link&sk=8545210a35874c43bdafd652213f1779